src/lib/y2storage/autoinst_profile/partition_section.rb

```diff
@@ -48,40 +48,41 @@ class PartitionSection < SectionWithAttributes
       NO_CREATE_IDS = []
       private_constant :NO_CREATE_IDS
 
-      # rubocop:disable Metrics/MethodLength
+      ATTRIBUTES = [
+        { name: :create },
+        { name: :filesystem },
+        { name: :format },
+        { name: :label },
+        { name: :uuid },
+        { name: :lv_name },
+        { name: :lvm_group },
+        { name: :mount },
+        { name: :mountby },
+        { name: :partition_id },
+        { name: :partition_nr },
+        { name: :partition_type },
+        { name: :subvolumes },
+        { name: :size },
+        { name: :crypt_fs },
+        { name: :loop_fs },
+        { name: :crypt_key },
+        { name: :raid_name },
+        { name: :raid_options },
+        { name: :mkfs_options },
+        { name: :fstab_options, xml_name: :fstopt },
+        { name: :subvolumes_prefix },
+        { name: :create_subvolumes },
+        { name: :resize },
+        { name: :pool },
+        { name: :used_pool },
+        { name: :stripes },
+        { name: :stripe_size, xml_name: :stripesize },
+        { name: :bcache_backing_for },
+        { name: :bcache_caching_for }
+      ].freeze
+
```

Comment:
I usually add a line like this right after the definition of the constant... to avoid temptations of programmers ignoring the `self.attributes` method and accessing the constant directly (which breaks the uniform access principle).

```ruby
private_constant :ATTRIBUTES
```